extension/resources/web-components-payment-type.json

```diff
@@ -0,0 +1,99 @@
+{
+  "key": "ctp-adyen-integration-web-components-payment-type",
```

Comment:
```suggestion
  "key": "commercetools-adyen-integration-payment-type",
```

Reply:
We have the other resources named with `ctp`. I would also leave the `web components` there in case we want to support a different approach in the future.